src/components/Cards/index.tsx

```diff
@@ -46,7 +46,7 @@ const Cards = ({ items, theme, text, ctaButtons }: CardsProps) => {
   const background = theme === 'dark' ? 'primary.dark' : 'background.paper';
   const textColor = theme === 'dark' ? 'primary.contrastText' : 'text.primary';
 
-  const isMasonry = !!text?.body && !!text?.subtitle;
+  const isMasonry = true;
```

Comment:
why this value has changed?
and why is it required if the value is `true`?

I mean, if we know for sure that the `isMasonry` variable has value `true`, then I wouldn't expect any condition on that, like `if (isMasonry) {/* some code here */}`.

so, @Meolocious, can you explain the reason behind this change?

Reply:
Changed this part to a new provided code. It was for the layout in masonry mode made by Salvatore but I managed it in another way to let it work